schema/revert/database_functions/add_missing_indices.sql

```diff
@@ -0,0 +1,7 @@
+-- Revert ggircs-portal:database_functions/add_missing_indices from pg
+
+BEGIN;
+
+-- XXX Add DDLs here.
+
+COMMIT;
```

Comment:
Do you want to write a revert?

Reply:
No, I don't really want to delete all the indices that were created, the deploy file is idempotent anyway. I'll add a comment here and to the verify file that they are not needed for this migration

Reply:
sounds good to me